web/src/cron.rs

```diff
@@ -0,0 +1,21 @@
+use std::time::Duration;
+
+use cja::cron::{CronRegistry, Worker};
+
+use crate::{jobs, AppState};
+
+fn hours(h: u64) -> Duration {
+    Duration::from_secs(h * 60 * 60)
+}
+
+fn cron_registry() -> CronRegistry<AppState> {
+    let mut registry = CronRegistry::new();
+
+    registry.register_job(jobs::sessions::Cleanup, hours(1));
```

Comment:
We only have a single job that we register on the Cron

Every hour we want to check the Sessions table for sessions that are more that 24 hours expired and delete them to keep the table nice and clean